buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -526,6 +526,96 @@
         "failOnStandardError": "false"
       }
     },
+	{
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Initialize docker - Debian 9",
```

Comment:
Look at DockerCommonRunArgs_Debian9 build variable. It does the volume mapping to prebuilt sharedframeworkpublish folder from binary build. Also, distro specific build steps do not contain binary build step. for e.g look at https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build?_a=summary&buildId=875149